pkg/bundle/verification_content.go

```diff
@@ -86,3 +94,39 @@ func (pk *PublicKey) GetCertificate() *x509.Certificate {
 func (pk *PublicKey) HasPublicKey() (verify.PublicKeyProvider, bool) {
 	return *pk, true
 }
+
+func (pk *PublicKey) GetCertificateChain() []*x509.Certificate {
+	return nil
+}
+
+func (cc *CertificateChain) CompareKey(key any, tm root.TrustedMaterial) bool {
+	if len(cc.Certificates) < 1 {
+		return false
+	}
+	return (&Certificate{cc.Certificates[0]}).CompareKey(key, tm)
+}
+
+func (cc *CertificateChain) ValidAtTime(t time.Time, tm root.TrustedMaterial) bool {
+	if len(cc.Certificates) < 1 {
+		return false
+	}
+	return (&Certificate{cc.Certificates[0]}).ValidAtTime(t, tm)
+}
+
+func (cc *CertificateChain) GetCertificate() *x509.Certificate {
```

Comment:
Minor: would `GetLeafCertificate()` be more clear?

Reply:
done